### ISSUE TITLE
Exclude Organization Members

### ISSUE THREAD

**Issue by emilyfeder** [jcua-tech/jcua-renxt#33]
Currently we are showing records with no first or last names. Those are apparently Organizational Members, as indicated by a specific Constituent Field (not sure which one). We would either need to get that field and filter server side on it, or we could just filter out any record that has no first or last name on the server side to shortcut that.

**Comment by emilyfeder** [comment]
There are is a constituent code = Organization and a constituent code = Individual, and we want to exclude the organizations. To grab the members list currently, we are using `constituent/v1/constituents?constituent_code=Member` However, we can't just add `&constituent_code=Individual` onto it because the query would look for [constituents with code = Member OR code = Individual ](https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/operations/ListConstituents). So I think the route is to do another query for the [constituent codes for all members](https://developer.sky.blackbaud.com/docs/services/56b76470069a0509c8f1c5b3/operations/ListConstituentCodesAllConstituents) and use that to filter the member list to only ones with constituent code = Organization.

**Comment by alexsoble** [comment]
I'm noticing data quality issues that will make implementing this feature difficult.

Clicking around the UI, I'm see things like: 

1. My own record is not marked as "Individual."
2. Searching for the keywords "Foundation" and "Endowment", I see results with Constituent Code values of "Organization", "Individual", "O", "NO", "PR"...

**Comment by alexsoble** [comment]
Some other ways we could implement this:

1. In the JSON, I'm noticing a `'type': 'Organization'` key/value pair.
2. In the original email, missing first or last names were mentioned as a sign that we have an "Organization."

**Comment by alexsoble** [comment]
tl;dr when looking for who's an Organization, the API JSON seems to have more valuable data than the UI. Not sure how to see constituent "type" yet in the UI, but it's not the same as the "Constituent Codes" you see at the top of each constituent's page in RENXT:

![constituent-codes](https://user-images.githubusercontent.com/3209501/54490982-b8853300-4888-11e9-84a7-88c72b253ddd.png)

**Comment by emilyfeder** [comment]
Great research and noticing `'type': 'Organization'` versus `'type':'Individual'` in the json! I think on the list comprehensions in `geocode_constituents_whose_data_change` and `fetch_geocoded_constituents` you can add an if statement `[...for member in members if (member.get('first') and member.get('last')]` or `[...for member in members if member.get('type') != 'Organization']` should work!

**Comment by alexsoble** [comment]
Closed by #40.
